### DISCUSSION TITLE
Temperature Issue?

### DISCUSSION THREAD

**Discussion by 4irm** [Blackymas/NSPanel_HA_Blueprint#2338] [Q&A] [answered]
I upgraded from 4.3.8 to 4.3.11 and incorporated the Bluetooth add-on as well. No I see the temperature 1-2 degrees Celsius higher before and I have the impression the panel is slightly warmer.

Any known issues, obviously this is a problem?

Thank you!

**Comment by 4irm** [comment, marked as answer]
For unknown reasons it normalized after 2 days...

**Comment by distante** [comment]
Interesting, I did the exact same thing right today and I also see the panel reporting warmer values.

@edwardtfn maybe when the Bluetooth chip is activated it gets warmer?

**Reply by edwardtfn** [reply]
This is likely the case. I remember some discussion around this in the past.

**Reply by distante** [reply]
To disable the Bluetooth proxy, we have to flash it without OTA, correct?

**Reply by edwardtfn** [reply]
Nope. You can use ota to flash it. Just remove the BLE related entries from your yaml.

**Reply by distante** [reply]
So from now on we can enable it and disable it with OTA? I ask because in the Readme says that to enable the Bluetooth proxy we have to flash it with USB.

**Reply by edwardtfn** [reply]
You must flash via serial to enable BLE and/or ESP-IDF, as that requires a different partitioning, but once you have that partitioning, it is fine to use it without BLE.

**Reply by distante** [reply]
Ah perfect thank you. I will wait one day more to see if (like the comment above said) it stabilizes, if no, I will probably disable the Bluetooth proxy until I have external (and accurate) temperature sensors.
